libr/core/cmd.c

```diff
@@ -4007,20 +4022,27 @@ R_API int r_core_cmd_foreach3(RCore *core, const char *cmd, char *each) { // "@@
 		}
 #endif
 		break;
-	case 's':
+	case 's': // XXX this command will crash when used with 'oc' (ocm'@@@s)
```

Comment:
this `./binr/radare2/radare2 -c "ocm'@@@s" /bin/ls` does not segfault for me, even with ASAN.

Reply:
this pls

Reply:
because its fixed